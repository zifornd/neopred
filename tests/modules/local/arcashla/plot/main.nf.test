nextflow_process {

    name "Test Process ARCASHLA_PLOT"
    script "modules/local/arcashla/plot/main.nf"
    process "ARCASHLA_PLOT"

    test("Should run without failures") {

        when {
            params {
                // define parameters here. Example:
                // outdir = "tests/results"
            }
            process {
                """
                // define inputs of the process here. Example:
                // input[0] = file("test-file.txt")
                """
            }
        }

        then {
            assert process.success
            assert snapshot(process.out).match()
        }

    }

}
