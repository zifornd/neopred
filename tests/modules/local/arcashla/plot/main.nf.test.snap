{
    "Should run without failures": {
        "content": [
            {
                "0": [
                    "hla_frequency_plot.pdf:md5,a65fa5b6742fd0da5fe0561e6e030582"
                ],
                "1": [
                    "versions.yml:md5,5c27e18ea51c19d4d196472fcb5ee468"
                ],
                "hla_plot": [
                    "hla_frequency_plot.pdf:md5,a65fa5b6742fd0da5fe0561e6e030582"
                ],
                "versions": [
                    "versions.yml:md5,5c27e18ea51c19d4d196472fcb5ee468"
                ]
            }
        ],
        "meta": {
            "nf-test": "0.9.2",
            "nextflow": "24.10.1"
        },
        "timestamp": "2025-01-06T09:27:21.442076"
    }
}